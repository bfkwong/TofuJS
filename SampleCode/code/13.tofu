map = {
        "1": 2, 
        "3": 4, 
        "5": {
          "6": 7
        },
        "7": [1,2]
      };

list = [1,2,3,4,5, {"1": 2}]; 

print(map["1"]);
print(map["5"]["6"]);

map["1"] = 10; 
map["5"]["6"] = 11; 

print(map["1"]);
print(map["5"]["6"]);

print(list[1]);
print(list[5]["1"]);

list[1] = 12; 
list[5]["1"] = 15; 

print(list[1]);
print(list[5]["1"]);

print(map["7"][0]);
map["7"][0] = 18; 
print(map["7"][0]);
