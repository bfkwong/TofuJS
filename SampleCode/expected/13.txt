2
7
10
11
2
2
12
15
1
18
